cytoscape-navigator.js

```diff
@@ -887,8 +880,9 @@
       }
 
       $img.style['position'] = 'absolute';
-      $img.style['left'] = translate.x + 'px';
-      $img.style['top'] = translate.y + 'px';
+      $img.style['width'] = w + 'px';
```

Comment:
I think this code, breaks the aspect ratio. It needs changes
![image](https://user-images.githubusercontent.com/8426741/74903955-93177800-53bb-11ea-8607-240317c516fd.png)

Reply:
I hope it is fixed with commit https://github.com/cytoscape/cytoscape.js-navigator/pull/43/commits/76c6aff40c9a9141f7c8b0c2651c0667c92ffd24